shell/packages/sandstorm-ui-topbar/topbar.js

```diff
@@ -159,8 +183,53 @@ Template.sandstormTopbar.events({
     Template.instance().data.closePopup();
   },
 
+  "click .toggle-sidebar": function (event) {
+    var topbar = Template.instance().data;
```

Comment:
Would it be possible to animate the sidebar-toggling, so that it doesn't happen instantaneously? I'm getting some whiplash when I click on it.